changelog.txt

```diff
@@ -1,6 +1,6 @@
 V.Next
 ----------
-- [MINOR] Deprecate Common logger wrappers (#2157)
+- [MINOR] Deprecate Common logger wrapper (#2157)
```

Comment:
do we still need this changeLog?

Reply:
Yeah, technically one other wrapper was deprecated